sbpy/data/ephem.py

```diff
@@ -143,11 +178,17 @@ def from_horizons(cls, targetids, id_type='smallbody',
             else:
                 all_eph = vstack([all_eph, eph])
 
-        # identify time scales returned by Horizons query
-        timescales = array(['UTC'] * len(all_eph))
-        timescales[all_eph['datetime_jd'] < 2437665.5] = 'UT1'
-        # according to Horizons documentation
-        all_eph.add_column(Column(timescales, name='timescale'))
+        # turn epochs into astropy.time.Time and apply timescale
+        # https://ssd.jpl.nasa.gov/?horizons_doc
+        all_eph['epoch'] = Time(all_eph['datetime_jd'], format='jd',
+                                scale='utc')
+        if any(all_eph['datetime_jd'] < 2437665.5):
+            print('pimmel')
+            all_eph['epoch'][all_eph['datetime_jd'] < 2437665.5] = (
+                Time(all_eph['datetime_jd'][all_eph['datetime_jd'] <
+                                            2437665.5],
+                     format='jd', scale='ut1'))
+        all_eph.remove_column('datetime_jd')
 
         if bib.status() is None or bib.status():
```

Comment:
This test is already addressed in `bib.register`, so no need to have it here.  It occurs 6 times in `sbpy.data`.